### ISSUE TITLE
升级后v5.4.1出现大量【Signal 11 was raised.】crash，无法有效复现，有同学遇到吗

### ISSUE THREAD

**Issue by csfuwwc** [aliyunvideo/AliPlayerSDK#62]
线上crash截图
![截屏2021-07-07 下午3 11 16](https://user-images.githubusercontent.com/7477885/124716188-5f001100-df36-11eb-91ec-8912fd98b301.png)

用户操作行为，都是点击播放器开始播放，但是无法有效复现
![截屏2021-07-07 下午3 19 18](https://user-images.githubusercontent.com/7477885/124716516-c1f1a800-df36-11eb-8a74-95af94ea512c.png)

v5.3.1升级到v5.4.1版本出现的这个问题，之前没有

**Comment by I-m-SuperMan** [comment]
有详细的日志和崩溃的文件？

**Comment by csfuwwc** [comment]
> 有详细的日志和崩溃的文件？

【Signal 11 was raised.】bugly没抓到，神策抓到了只有个这个crash信息，除了这个没有其他信息。。

  我先降回到v5.3.1，跟进用户看看没有有其他必现路径叭

**Comment by csfuwwc** [comment]
> 有详细的日志和崩溃的文件？

bugly没抓到这个crash，神策只有【Signal 11 was raised.】这个信息。。

猜测Xcode里crash最高的就是这个，看起来是kvo释放问题

Date/Time:           2021-07-07 09:09:05.8281 +0800
Launch Time:         2021-07-07 09:06:01.0737 +0800
OS Version:          iPhone OS 14.6 (18F72)
Release Type:        User
Baseband Version:    2.05.01
Report Version:      104

Exception Type:  EXC_BAD_ACCESS (SIGSEGV)
Exception Subtype: KERN_INVALID_ADDRESS at 0x000000000000001c
VM Region Info: 0x1c is not in any region.  Bytes before following region: 4330438628
      REGION TYPE                 START - END      [ VSIZE] PRT/MAX SHRMOD  REGION DETAIL
      UNUSED SPACE AT START
--->  
      __TEXT                   1021d4000-1021d8000 [   16K] r-x/r-x SM=COW  ...DianZhongChou

Triggered by Thread:  0

Thread 0 name:
Thread 0 Crashed:
0   libobjc.A.dylib               	0x00000001b6e32290 object_isClass + 16 (objc-runtime-new.h:495)
1   Foundation                    	0x00000001a36f3ebc KVO_IS_RETAINING_ALL_OBSERVERS_OF_THIS_OBJECT_IF_IT_CRASHES_AN_OBSERVER_WAS_OVERRELEASED_OR_SMASHED + 48 (NSKeyValueObserving.m:1131)
2   Foundation                    	0x00000001a36f1374 NSKeyValueWillChangeWithPerThreadPendingNotifications + 272 (NSKeyValueObserving.m:1162)
3   QuartzCore                    	0x00000001a573de84 CA::Layer::begin_change(CA::Transaction*, unsigned int, objc_object*, objc_object*&) + 248 (CALayer.mm:3392)
4   QuartzCore                    	0x00000001a574d6c8 CA::Layer::set_delegate(objc_object*, bool) + 144 (CALayer.mm:5936)
5   UIKitCore                     	0x00000001a52c8154 _UIViewLayerSetView + 40 (UIView_NonARC.m:37)
6   UIKitCore                     	0x00000001a528e4ec -[UIView dealloc] + 632 (UIView.m:481)
7   libobjc.A.dylib               	0x00000001b6e5457c AutoreleasePoolPage::releaseUntil(objc_object**) + 204 (NSObject.mm:944)
8   libobjc.A.dylib               	0x00000001b6e5441c objc_autoreleasePoolPop + 212 (NSObject.mm:1211)
9   CoreFoundation                	0x00000001a239af3c _CFAutoreleasePoolPop + 32 (NSObject.m:798)
10  CoreFoundation                	0x00000001a230761c __CFRunLoopPerCalloutARPEnd + 48 (CFRunLoop.c:762)
11  CoreFoundation                	0x00000001a23022b4 __CFRunLoopRun + 2592 (CFRunLoop.c:3120)
12  CoreFoundation                	0x00000001a2301360 CFRunLoopRunSpecific + 600 (CFRunLoop.c:3242)
13  GraphicsServices              	0x00000001b993f734 GSEventRunModal + 164
14  UIKitCore                     	0x00000001a4d7c584 -[UIApplication _run] + 1072 (UIApplication.m:3269)
15  UIKitCore                     	0x00000001a4d81df4 UIApplicationMain + 168 (UIApplication.m:4740)
16  MoDianZhongChou               	0x0000000103689210 main + 21713424 (main.m:14)
17  libdyld.dylib                 	0x00000001a1fbdcf8 start + 4

**Comment by I-m-SuperMan** [comment]
这个崩溃看着跟播放器没什么太大的关系。不确定是播放器的问题。

**Comment by csfuwwc** [comment]
> 这个崩溃看着跟播放器没什么太大的关系。不确定是播放器的问题。

对。日志无法定位到具体代码，目前只知道crash都是点击播放器播放按钮时产生的。。然而手动至今没有复现，我先试试把播放器回退v5.3.1版本看看[哭]

**Comment by csfuwwc** [comment]
> 这个崩溃看着跟播放器没什么太大的关系。不确定是播放器的问题。

找到手动复现步骤了~怀疑是player内部代码逻辑


[18:39:51] -[AliyunVodPlayerView onPlayerStatusChanged:oldStatus:newStatus:] [第1269行] 播放器状态更新：播放
[18:39:51] -[AliyunVodPlayerView onPlayerEvent:eventType:] [第1044行] 播放器:首帧加载完成封面隐藏
2021-07-07 18:59:37.987968+0800 MoDianZhongChou[61479:11847524] *** -[SampleDisplayLayerRender retain]: message sent to deallocated instance 0x137b71110

**Comment by csfuwwc** [comment]
用户反馈了，和复现操作基本一致。v5.4.1播放器在拖动进度条后，pause/stop和release操作大几率闪退，v5.3.1没有问题
